_posts/2021-05-26-cockpit-245.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Cockpit 245
+author: lis
+date: 2021-05-26
+tags: cockpit linux
+slug: cockpit-245
+category: release
+summary: Cockpit with metrics and storage improvements
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 245 and cockpit-podman version 31.
+
+## Release process: No more cockpit-cache tarball
```

Comment:
```suggestion
## Release: Drop cockpit-cache tarball
```
I'd suggest moving this to the bottom, as most users won't ever have heard about this, nor care.

Reply:
Good point. I always think about distro packagers when I'm writing these, but it's actually users I should be thinking about...